icd/api/compiler_solution_llpc.cpp

```diff
@@ -567,6 +567,12 @@ VkResult CompilerSolutionLlpc::CreateLlpcCompiler()
         llpcOptions[numOptions++] = "-debug";
     }
 
+    // Use LLPC relocatable shaders when requested
+    if (settings.enableRelocatableShaders)
+    {
+        llpcOptions[numOptions++] = "-use-relocatable-shader-elf";
```

Comment:
As discussed earlier today, we think this should be done by a new field in PipelineOptions (in vkgcDefs.h), rather than adding a new setting of a command-line option here. We're trying to remove setting codegen-affecting command-line options in the driver.

I had a bad connection earlier, so I think I missed the gist of what you were saying. Were you saying you are happy to go off and do the PipelineOptions thing? Or that you want to get this change in as-is first to continue testing, then go off and do the "proper" fix?

Reply:
I'll make the change right away.  No need to merge this right away.